recipes/deprecated/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "Deprecated" %}
+{% set version = "1.2.0" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "42b5dffc4023401148e9d2757f46a4a9fcccf114547a4620347ae45f9e77a21c" %}
```

Comment:
Should hash_type and hash_value be moved directly under the `source` section?

Reply:
probably best to just keep close to what conda skeleton spit out since that is one of the major things that the tick bot hits

Reply:
@CJ-Wright, can you clarify what is the optimal way to write the SHA256 sums? It looks like `skeleton` and `example` are conflicting with each other.

Reply:
That's correct. If there are multiple sources, the bot doesn't know which checksum the jinja2 variable is for, so it might replace it incorrectly. In this recipe where there is only one checksum, there is no ambiguity so the bot should be able to replace the jinja variable correctly.

It might be good practice to get rid of the jinja variables for hash_type and hash_value though, since there are certain cases where that breaks the bot and it doesn't seem like there's much of a reason to use the variables since they are only used once. I think @ocefpaf changed the example to remove jinja variables for checksum, but skeleton wasn't updated yet.